vms/platformvm/block/executor/proposal_block_test.go

```diff
@@ -546,10 +535,7 @@ func TestBanffProposalBlockUpdateStakers(t *testing.T) {
 			advanceTimeTo: []time.Time{staker1.startTime, staker2.startTime, staker3.startTime, staker5.startTime},
 			expectedStakers: map[ids.NodeID]stakerStatus{
 				staker1.nodeID: current,
-
-				// given its txID, staker2 will be
-				// rewarded and moved out of current stakers set
-				// staker2.nodeID: current,
+				staker2.nodeID: current,
```

Comment:
Why is this being changed?

Reply:
Hey @dhrubabasu thanks for spotting this.
This is another way this test is brittle: it asserts that stakers are there if the right chain time arrives, but it does not work with stakers removal. Specifically in this test case, we reach the chain-time where staker5 should be added (which happens and it is duly checked among test assertion) *and* staker2 should be remove, which does not happen.
In the interest of scoping this PR, I carried out the minimal number of changes. Will make another PR to fix/improve this test!